src/templates/components/staffProfileTeaser/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { StaffProfileTeaser as FormattedStaffProfileTeaser } from '@/types/formatted/staffProfile'
+import { MediaImage } from '@/templates/common/mediaImage'
+import { PhoneNumber } from '@/templates/common/phoneNumber'
+
+export const StaffProfileTeaser = ({
+  media,
+  firstName,
+  lastName,
+  suffix,
+  description,
+  vamcTitle,
+  phoneNumber,
+  link,
+  id,
+}: FormattedStaffProfileTeaser) => {
+  return (
+    <div className="vads-u-display--flex vads-u-margin-bottom--4">
+      {media && (
+        <div
+          className="vads-u-margin-right--2 vads-u-flex--auto"
+          data-testid="staff-profile-image"
+        >
+          <MediaImage
+            {...media}
+            className="person-profile-detail-page-image vads-u-width--auto"
```

Comment:
Looks like in prod these use a different custom class that puts a different max height (smaller) on the images